BootloaderCorePkg/BootloaderCorePkg.dec

```diff
@@ -133,7 +133,8 @@
   gPlatformModuleTokenSpaceGuid.PcdSmbiosTablesSize       |     0x1000 | UINT16 | 0x200000E3
 
   # Size of the Hash store allocated in bootloader
-  gPlatformModuleTokenSpaceGuid.PcdHashStoreSize       | 0x00000200 | UINT32 | 0x200000F1
+  gPlatformModuleTokenSpaceGuid.PcdHashStoreSize          | 0x00000200 | UINT32 | 0x200000F1
+  gPlatformModuleTokenSpaceGuid.PcdFastBootEnabled        |  FALSE     | BOOLEAN| 0x200000F2

```

Comment:
Add a comment on the expected behavior when this PCD is enabled

Reply:
ok

Reply:
Can we have a better name to avoid another "fastboot" term?

Reply:
sorry, I had merged the patch for platform patch update. do you mean "MrcFastBoot" from FSP UPD?